lib/order.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'menu'
```

Comment:
Great use of SRP, each method is doing as little as possible, and delegating where needed!